.github/workflows/test.yml

```diff
@@ -133,43 +153,31 @@ jobs:
       - name: Run fluent-postgres-kit tests
         run: swift test --package-path fluent-postgres-driver
 
-  # Run macOS unit tests against various configurations
   macos-unit:
     if: ${{ !(github.event.pull_request.draft || false) }}
     strategy:
       fail-fast: false
       matrix:
-        dbimage: ['postgresql@14']
-        dbauth: ['scram-sha-256']
-        macos: ['macos-13']
-        xcode: ['latest-stable']
-    runs-on: ${{ matrix.macos }}
+        xcode-version:
+          - '~14.3'
+          - '15.0-beta'
+    runs-on: macos-13
     env:
       POSTGRES_HOSTNAME: 127.0.0.1
       POSTGRES_DB: postgres
     steps:
       - name: Select latest available Xcode
         uses: maxim-lobanov/setup-xcode@v1
         with:
-          xcode-version: ${{ matrix.xcode }}
+          xcode-version: ${{ matrix.xcode-version }}
       - name: Install Postgres, setup DB and auth, and wait for server start
-        env:
-          POSTGRES_VERSION: ${{ matrix.dbimage }}
-          POSTGRES_AUTH_METHOD: ${{ matrix.dbauth }}
         run: |
-          export PATH="$(brew --prefix)/opt/${POSTGRES_VERSION}/bin:$PATH" PGDATA=/tmp/vapor-postgres-test
-          (brew unlink postgresql || true) && brew install "${POSTGRES_VERSION}" && brew link --force "${POSTGRES_VERSION}"
-          initdb --locale=C --auth-host "${POSTGRES_AUTH_METHOD}" -U "${POSTGRES_USER}" --pwfile=<(echo "${POSTGRES_PASSWORD}")
+          export PATH="$(brew --prefix)/opt/postgresql@14/bin:$PATH" PGDATA=/tmp/vapor-postgres-test
```

Comment:
why `@14`? Isn't the CI using postgres 15 now?

Reply:
The Linux unit tests cover Postgres versions 11, 13, and 15; we take the macOS tests as an excuse to cover 14 as well.